src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.cs

```diff
@@ -72,6 +108,17 @@ private void CheckStart()
             }
         }
 
+        private static IPAddress GetAddressSnapshot(IPAddress address)
+        {
+            IPAddress addressSnapshot = (address.AddressFamily == AddressFamily.InterNetwork) ?
+#pragma warning disable CS0618 // IPAddress.Address is obsoleted, but it's the most efficient way to get the Int32 IPv4 address
```

Comment:
I personally think that we should avoid using obsoleted APIs. How significant of a performance difference do you expect?

Reply:
I agree but this is what was there already.  Should I look into changing this?

Reply:
I think it'd be worthwhile to look into. If there is a measurable performance hit I'm not opposed to keeping it as is though.

Reply:
No problem, I'll look into it.

Reply:
The deprecated call is perhaps up to 4x faster than the alternative (on my machine) HOWEVER I think this is little more than academic.  I'm having to measure to the level of ticks to actually see a difference and I doubt this method is going to be used enough to justify the use of the deprecated property.

I will submit a new commit but can always revert if you feel it's necessary.